pkgs/applications/video/vokoscreen-ng/default.nix

```diff
@@ -57,7 +53,12 @@ mkDerivation rec {
   '';
 
   postInstall = ''
+    mkdir -p $out/bin $out/share/applications $out/share/icons
+    cp ./vokoscreenNG $out/bin/
+    cp ./src/applications/vokoscreenNG.desktop $out/share/applications/
+    cp ./src/applications/vokoscreenNG.png $out/share/icons/
     qtWrapperArgs+=(--prefix GST_PLUGIN_SYSTEM_PATH_1_0 : "$GST_PLUGIN_SYSTEM_PATH_1_0")
+    wrapQtApp $out/bin/vokoscreenNG
```

Comment:
Is this now double wrapping?

Reply:
No, if I don't do it manually  it's not detecting the binary not sure why (I even had to copy the binary by hand).
It screams a 
```sh
/bin: Is a directory
make: Nothing to be done for 'install'.
```